### ISSUE TITLE
Admin color schemes fail color accessibility

### ISSUE THREAD

**Issue by niranjan-uma-shankar** [Automattic/wp-calypso#101054]
### Context and steps to reproduce

I reported this issue in Slack: p1739544516924279/1739470190.013939-slack-C029GN3KD. 

Many of our admin color schemes fail the color accessibility test. Some of the color combinations are difficult to read even for those without colour accessibility requirements. It appears that some of these may be intentional (to have low contrast schemes), but do we need so many colour schemes to have low contrast and fail accessibility?

Check the "Launch site" button on the "Light" colour scheme.

<img width="1102" alt="Image" src="https://github.com/user-attachments/assets/4de99a39-1045-40a3-9509-8149ecaac9c8" />

A few examples of color accessibility failures reported in Chrome dev tools. These are just some examples, we'd need to comb through every admin colour scheme to comprehensively fix these issues.

![Image](https://github.com/user-attachments/assets/3c709dfa-d54a-486b-bb8a-ea6c2e4f7eab)
![Image](https://github.com/user-attachments/assets/bda1fe27-370e-4c7b-b62a-a7381a8b709d)
![Image](https://github.com/user-attachments/assets/1eee35fd-7b6c-4faa-812d-b22d71fb9513)
![Image](https://github.com/user-attachments/assets/53d22fab-fd72-41a4-b632-01b2271fb063)

### Site owner impact

Between 20% and 60% of the total website/platform users

### Severity

Moderate

**Comment by janmckell** [comment]
Hi @Automattic/dotcom-design ! Is this an issue you'd be able to take a look at and add to your tasks?

**Comment by jasmussen** [comment]
Most of these color schemes do not pass contrast ratios in the core software itself, which is an argument to retire and revisit those schemes in a future version. It also means this is not something we can or should fix no the dotcom side of things, i.e. not a bug. The recommendation would be to use the default color scheme. CC: @lsl as you worked on adjacent stuff: do you agree?

However the "Launch" button is new, and likely uses a hard-coded spot color instead of the color-scheme inherited default. This one we can/should fix. @richtabor @nuriapenya I think you may have worked on pieces related to this.

**Comment by lsl** [comment]
Fix should be upstream, or maybe happen after we remove the non core ones.

Launch site button needs work on the wp-admin side. In the last round of testing I noticed the icon alignment is a bit low and color schemes aren't implemented, we'll pick these up after the color studio update is figured out.

cc @Automattic/lego
